contrib/nodejsscan/eval_vm_injection.yaml

```diff
@@ -47,24 +47,6 @@ rules:
         $CONTEXT = {$NAME: <... $VAR ...>};
         ...
         $VM.runInContext($CODE,<... $CONTEXT ...>,...);
-    - pattern: |
```

Comment:
cc @ajinabraham the semgrep metachecker found a few "duplicate patterns" errors in nodejsscan rules.
Not sure if because you did too many copy paste or because those copy-paste should have been
extended.

Reply:
@aryx We're actually pulling these `contrib` rules out soon; we ingest the latest rules from @ajinabraham's repository directly. Anything that's not in `contrib` is fair game, though 👍